package.json

```diff
@@ -6,9 +6,11 @@
   "version": "1.3.2",
   "dependencies": {
     "commander": "^2.6.0",
+    "debug": "^2.1.1",
     "denodeify": "^1.2.0",
     "es6-promise": "^2.0.1",
     "haikro": "^1.9.0",
+    "heroku-client": "^1.9.1",
```

Comment:
I've had pretty poor experiences using the heroku client. No promises, does weird things around pagination… generally felt like it was adding more complexity than it saved.  If you don't mind me switching your new code to just make the raw api call I'd be happy to do it…